packages/patternfly-4/react-core/src/components/AboutModal/AboutModal.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/Backdrop/backdrop.css';
+import { canUseDOM } from 'exenv';
+import { KEY_CODES } from '../../helpers/constants';
+import { AboutModalContainer } from './helpers/AboutModalContainer';
+
+export interface AboutModalProps {
+  /** Content rendered inside the about modal  */
+  children: React.ReactNode;
+  /** Additional classes added to the about modal  */
+  className?: string;
+  /** Flag to show the about modal  */
+  isOpen?: boolean;
+  /** A callback for when the close button is clicked  */
+  onClose?: () => void;
+  /** Product name  */
+  productName?: string;
+  /** Trademark information  */
+  trademark?: string;
+  /** The URL of the image for the brand  */
+  brandImageSrc: string;
+  /** The alternate text of the brand image  */
+  brandImageAlt: string;
+  /** The URL of the image for the background  */
+  backgroundImageSrc?: string;
+  /** Prevents the about modal from rendering content inside a container; allows for more flexible layouts  */
+  noAboutModalBoxContentContainer?: boolean;
+};
+
+interface AboutModalState {
+  ariaLabelledBy: string;
+  ariaDescribedBy: string;
+  container: HTMLElement;
+}
+
+export class AboutModal extends React.Component<AboutModalProps, AboutModalState> {
+  private static currentId = 0;
+  static defaultProps = {
+    className: '',
+    isOpen: false,
+    onClose: (): any => undefined,
+    productName: '',
+    trademark: '',
+    backgroundImageSrc: '',
+    noAboutModalBoxContentContainer: false
+  };
+
+  constructor(props: AboutModalProps) {
+    super(props);
+    this.state = {
+      container: undefined,
+      ariaLabelledBy: `pf-about-modal-title-${AboutModal.currentId++}`,
```

Comment:
Can you remove these from the state.  They are not stateful.

Reply:
I'll use class attributes instead.